boilerplate/ignite/templates/model/NAME.test.ts.ejs

```diff
@@ -1,3 +1,6 @@
+---
+destinationDir: app/models
```

Comment:
Not a huge fan of `destinationDir` as the name. IMO, `dir` is a much shorter option (which matters in terminal commands) that still is sufficiently clear what's happening.

```
npx ignite-cli@latest g model user --dir=app/models
```

Reply:
That's fair, I was just matching what we had as the front matter prop to make things consistent but I like the shorter idea

Reply:
Updated